Appsilon/mbaza
# fix: clean state of the filters after importing a new data

Closes #193 

### Description
after importing a new data, filters are resetted and metrics display values for not filtered data

### How to test
- change filters for some imported data, notice how metrics values changed
- import the same csv file and notice that filters got resetted and metrics values returned to the initial state
